### ISSUE TITLE
windows client: installer fails due to netsh, etc., not in PATH

### ISSUE THREAD

**Issue by danyhm** [Jigsaw-Code/outline-client#278]
I ran into connection problem after updating to the latest version. it said couldn't configure routing tables. then I tried installing using the latest daily update release and it gives me this error.

![image](https://user-images.githubusercontent.com/21024667/45594855-ea3f3f00-b9b6-11e8-9ff1-5e31bde5a579.png)

**Comment by trevj** [comment]
@danyhm Thanks for trying Outline and many thanks for this report. I'm not sure how this could happen: `netsh` and `wmic` should be built into every version of Windows we support (7+). Can you please tell me:
- which version of Windows you're using
- do you have any virus checker and/or anti-malware installed?

**Comment by trevj** [comment]
@danyhm It looks like you're using 1.2.11, is that right?

**Comment by danyhm** [comment]
@trevj 
Thanks for the reply. I'm using windows 7 SP1 and I've been using outline for a couple of months now and it worked well so far until this update. yes i have Kaspersky Internet Security installed but didn't have any problem with it in the past.

and yes I'm using 1.2.11

**Comment by trevj** [comment]
@danyhm Good to hear Outline has been working well. We've changed quite a few things with the latest version. This kind of feedback is invaluable.

If you're on Windows 7, can you please tell me the value of your PATH environment variable? You can check this with:
- opening Control Panel
- find **View Advanced system settings**
- click **Environment variables...**
- under **System variables**, double-click `Path`

Thanks again!

**Comment by danyhm** [comment]
@trevj 
sorry for the late reply, here is the PATH variable of my PC

`C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\STMicroelectronics\st_toolset\asm;D:\Program Files (x86)\Intel\MPI-RT\5.0.2.044\intel64\bin;D:\Program Files\LLVM\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\STMicroelectronics\STM32 ST-LINK Utility\ST-LINK Utility;C:\Program Files (x86)\Gpg4win\..\GnuPG\bin`

**Comment by trevj** [comment]
@danyhm Many thanks! `netsh`, etc., are usually found at `c:\windows\system32` and this tells me that something has removed that directory from your system's PATH. I think if you add that directory to the PATH then Outline will work for you.

**Comment by trevj** [comment]
On our side, I think we could add c:\windows\system32 to the PATH used by the installer shell script(s).

**Comment by danyhm** [comment]
@trevj 
thanks for the support
I did as you said and ran the installer again. this time it gave an error due to 'wmic' not being recognized. so I googled and added `%SystemRoot%\System32\Wbem` to the path as well and ran the installer again.
then it progressed further and a TAP driver installation poped up and I clicked install but then it failed with this error
![outline](https://user-images.githubusercontent.com/21024667/45710663-870f0180-bb9c-11e8-9c5c-2e5c0f38db12.png)

then I ran the installer again just to make sure that error persists, but the error message changed to this

![outline2](https://user-images.githubusercontent.com/21024667/45710690-9a21d180-bb9c-11e8-9e18-43fd5f98dcb4.png)

so this time I added `%SYSTEMROOT%\System32\WindowsPowerShell\v1.0\` to the path and ran the installer again and it finished without any error.

and then I ran outline and tried connecting but it gave this error

![outline3](https://user-images.githubusercontent.com/21024667/45710738-b3c31900-bb9c-11e8-8675-d5fc430e66c2.png)

while I don't have any other VPN running.

this is all strange it was working all fine and I'm pretty sure I didn't change any system settings.
